index.js

```diff
@@ -32,16 +34,18 @@ function findVcvarsall() {
 
     // If that does not work, try the standard installation locations,
     // starting with the latest and moving to the oldest.
-    for (const ver of VERSIONS) {
-        for (const ed of EDITIONS) {
-            path = `${PROGRAM_FILES_X86}\\Microsoft Visual Studio\\${ver}\\${ed}\\VC\\Auxiliary\\Build\\vcvarsall.bat`
-            core.info(`Trying standard location: ${path}`)
-            if (fs.existsSync(path)) {
-                core.info(`Found standard location: ${path}`)
-                return path
-            }
-        }
-    }
+	  for (const prog_files of PROGRAM_FILES) {
+			for (const ver of VERSIONS) {
+					for (const ed of EDITIONS) {
+							path = `${prog_files}\\Microsoft Visual Studio\\${ver}\\${ed}\\VC\\Auxiliary\\Build\\vcvarsall.bat`
+							core.info(`Trying standard location: ${path}`)
+							if (fs.existsSync(path)) {
+									core.info(`Found standard location: ${path}`)
+									return path
+							}
+					}
+			}
+		}
```

Comment:
Hi, please follow the original rule of indentation.

```suggestion
    for (const prog_files of PROGRAM_FILES) {
        for (const ver of VERSIONS) {
            for (const ed of EDITIONS) {
                path = `${prog_files}\\Microsoft Visual Studio\\${ver}\\${ed}\\VC\\Auxiliary\\Build\\vcvarsall.bat`
                core.info(`Trying standard location: ${path}`)
                if (fs.existsSync(path)) {
                    core.info(`Found standard location: ${path}`)
                    return path
                }
            }
        }
    }
```